stream-chat-android-ui-components/src/main/res/values/dimens.xml

```diff
@@ -37,6 +41,7 @@
     <dimen name="stream_ui_channel_list_item_option_icon_width">50dp</dimen>
     <dimen name="stream_ui_channel_item_progress_bar_size">16dp</dimen>
     <dimen name="stream_ui_channel_list_item_loading_height">90dp</dimen>
+    <dimen name="stream_ui_channel_header_separator_height">1dp</dimen>
```

Comment:
Added this as a new dimension resource, as we were using the corresponding one in the old module in the new UI sample in a lot of places.